cmake/OusterSDKConfig.cmake.in

```diff
@@ -6,12 +6,12 @@ include(CMakeFindDependencyMacro)
 find_dependency(Eigen3)
 find_dependency(jsoncpp)
 find_dependency(CURL)
+find_dependency(spdlog)
 
 # viz dependencies
 if(@BUILD_VIZ@)
   set(OpenGL_GL_PREFERENCE GLVND)
   find_dependency(OpenGL)
-  find_dependency(Threads)
```

Comment:
I removed this while working on the logging story .. did anyone actually verified `viz` has not been affected?

Reply:
but wouldn't mind someone else checking ;)

Reply:
I will check